src/test/java/org/jenkinsci/plugins/workflow/libs/ResourceStepTest.java

```diff
@@ -71,7 +71,7 @@ public class ResourceStepTest {
         initFixedContentLibrary();
         
         LibraryConfiguration libraryConfig =  new LibraryConfiguration("stuff", new SCMSourceRetriever(new GitSCMSource(null, sampleRepo.toString(), "", "*", "", true)));
-        libraryConfig.setCachingConfiguration(new LibraryCachingConfiguration(0, ""));
+        libraryConfig.setCachingConfiguration(new LibraryCachingConfiguration(0, "",""));
```

Comment:
```suggestion
        libraryConfig.setCachingConfiguration(new LibraryCachingConfiguration(0, "", ""));
```

Reply:
nitpicking

Reply:
Updated in [e0a2432](https://github.com/jenkinsci/pipeline-groovy-lib-plugin/pull/16/commits/e0a243267abfdd351c1610b68985c11cea7eb8be)